ops/_private/harness.py

```diff
@@ -3421,39 +3433,52 @@ def add_layer(
                 info = pebble.CheckInfo(
                     name,
                     level=check.level,
+                    startup=check.startup,
                     status=status,
+                    threshold=3 if check.threshold is None else check.threshold,
```

Comment:
Can we make this magic number a named (non-public) constant, as it's used in a couple of places now?